I took a look at the Kokoro test failures for this PR. 
It looks like the `gslib.tests.test_hashing_helper.TestHashConfig` test suite is failing. Specifically, `testGetDownloadHashAlgs` expects a `CommandException` when an invalid `check_hashes` config is provided, but it isn't raising that exception (or it's raising a different exception), causing an `AssertionError`. 
Here is the traceback from the logs:
```python
FAIL: testGetDownloadHashAlgs (gslib.tests.test_hashing_helper.TestHashConfig.testGetDownloadHashAlgs)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmpfs/src/github/src/gsutil/gslib/tests/test_hashing_helper.py", line 365, in testGetDownloadHashAlgs
    self.fail('Expected CommandException for invalid check_hashes config.')
AssertionError: Expected CommandException for invalid check_hashes config.

--- a/gslib/tests/test_hashing_helper.py
+++ b/gslib/tests/test_hashing_helper.py
@@ -360,9 +360,10 @@ class TestHashConfig(testcase.GsUtilUnitTestCase):
 
     # Test misconfigured option raises CommandException
     with SetBotoConfigForTest([('GSUtil', 'check_hashes', 'invalid_option')]):
-      try:
-        hashing_helper.GetDownloadHashAlgs(self.logger, consider_crc32c=True)
-        self.fail('Expected CommandException for invalid check_hashes config.')
-      except CommandException as e:
-        self.assertIn('option is misconfigured', str(e))
+      with mock.patch.object(hashing_helper, 'UsingCrcmodExtension', return_value=False):
+        try:
+          hashing_helper.GetDownloadHashAlgs(self.logger, consider_crc32c=True)
+          self.fail('Expected CommandException for invalid check_hashes config.')
+        except CommandException as e:
+          self.assertIn('option is misconfigured', str(e))
 

